frontend/src/components/widgets/inputs/NumberInput.vue

```diff
@@ -162,11 +162,13 @@ export default defineComponent({
 					this.updateValue(this.value * directionMultiplier);
 				},
 				Callback: (): void => {
-					if (direction === "Increase" && this.incrementCallbackIncrease) this.incrementCallbackIncrease();
-					if (direction === "Decrease" && this.incrementCallbackDecrease) this.incrementCallbackDecrease();
+					if (direction === "Increase") this.incrementCallbackIncrease?.();
+					if (direction === "Decrease") this.incrementCallbackDecrease?.();
 				},
 				None: (): void => undefined,
-			}[this.incrementBehavior]());
+			};
+			const action = actions[this.incrementBehavior];
```

Comment:
Big win of readability over the old version. I remember going huh when first reading it